R/workflow_map.R

```diff
@@ -208,13 +209,52 @@ allowed_fn <-
   tibble::tibble(
     func = c(
       "tune_grid", "tune_bayes", "fit_resamples", "tune_race_anova",
-      "tune_race_win_loss", "tune_sim_anneal"
+      "tune_race_win_loss", "tune_sim_anneal", "tune_cluster"
     ),
-    pkg = c(rep("tune", 3), rep("finetune", 3))
+    pkg = c(rep("tune", 3), rep("finetune", 3), "tidyclust")
   )
 allowed_fn_list <- paste0("'", allowed_fn$func, "'", collapse = ", ")
 # nocov end
 
+# ---------------------------------------------
+check_object_fn <- function(object, fn, call = rlang::caller_env()) {
+   wf_specs <- purrr::map(object$wflow_id, ~extract_spec_parsnip(object, id = .x))
```

Comment:
Minor nitpick, this line is a little long

```suggestion
   wf_specs <- purrr::map(
     object$wflow_id, ~extract_spec_parsnip(object, id = .x)
   )
```